server/syncSubs.ts

```diff
@@ -54,20 +58,37 @@ async function syncSubscribers() {
     uid: uidMap.get(emailMap.get(sub.customer)),
   }));
 
+  // generate hash value over all current sub uids
+  currentSubsHash = 0;
+  for (let i = 0; i < result.length; i++) {
+    const row = result[i];
+    currentSubsHash = currentSubsHash ^ stringToHash(row.uid);
+  }
+
   // Upsert to DB
   // console.log(result);
-  try {
-    await postgres2?.query('BEGIN TRANSACTION');
-    await postgres2?.query('DELETE FROM subscriber');
-    for (let i = 0; i < result.length; i++) {
-      const row = result[i];
-      await insertObject(postgres2, 'subscriber', row);
+  if (currentSubsHash != lastSubsHash) {
+    try {
+      await postgres2?.query('BEGIN TRANSACTION');
+      await postgres2?.query('DELETE FROM subscriber');
+      await postgres2?.query('UPDATE room SET "isSubRoom" = false');
```

Comment:
is this false needed?  we're clearing the table on every run

Reply:
I thought this is only clearing the subscriber table and not the room table?

Reply:
oh I see. Sorry I'm tired and can't review code properly haha